tests/pytests/test_json_multi_numeric.py

```diff
@@ -286,6 +286,14 @@ def testInvertedIndexMultipleBlocks(env):
     expected_docs = ['doc:{}'.format(i) for i in chain(range(1, overlap + 1), range(doc_num - overlap + 1, doc_num + 1))]
     env.assertEqual(toSortedFlatList(res[1:]),toSortedFlatList(expected_docs), message='FT.SEARCH')
 
+def ft_info_to_dict(env, idx):
+    res = env.execute_command('ft.info', idx)
+    return {res[i]: res[i + 1] for i in range(0, len(res), 2)}
+
+def check_empty(env, idx, mem_usage):
+    d = ft_info_to_dict(env, idx)
+    env.assertEqual(float(d['num_records']), 0)
+    env.assertGreaterEqual(mem_usage, float(d['inverted_sz_mb']))
```

Comment:
those functions repeat themselves.

Reply:
`ft_info_to_dict()` was removed, it was replaced by `index_info()` from `common.py`
`check_empty()` was removed, it was replaced by a simple assertion